scripts/us_census/pep/pep_by_srh/input_url.json

```diff
@@ -0,0 +1,262 @@
+[
+    {
+        "download_path": "https://www2.census.gov/programs-surveys/popest/datasets/1990-2000/counties/asrh/co-99-10.txt",
+        "file_path": "1990_2000/county/"
+    },
+    {
+        "download_path": "https://www2.census.gov/programs-surveys/popest/datasets/2020-2023/counties/asrh/cc-est2023-alldata.csv",
```

Comment:
Remove this link as it should be dynamically constructed using the future year URL.

Reply:
Done.